src/main/java/com/softserve/demo/util/ContractMaker.java

```diff
@@ -85,13 +93,24 @@ private void setContractFields(Order order, AcroFields fields) throws IOExceptio
                 Service::getServiceName).collect(Collectors.joining("\n")));
     }
 
-    private String getContractName(Order order) {
+    private String getContractFileName(Order order) {
         return (order.getCustomer().getUser().getId() + "_"
                 + order.getProvider().getUser().getId() + "_" + order.getId() + ".pdf");
     }
 
-    private boolean createFolderIfNoExist(String folderName) {
-        File newDirectory = new File(folderName);
-        return (newDirectory.mkdir() || newDirectory.isDirectory());
+    private void createContractFolder(String folderName) {
+        String contractFolder = PATH + SEPARATOR + folderName;
+        this.fileStorageLocation = Paths.get(contractFolder).toAbsolutePath().normalize();
+        try {
+            Files.createDirectories(this.fileStorageLocation);
+        } catch (IOException e) {
+            log.error("Fail to create contract folder: " + e.getMessage());
+            throw new NotFoundException("Contract folder not found");
```

Comment:
Not found exception when something whent wrong wile creating directories? It should be something else for sure, coz this does not make sence to me